war/src/main/java/io/lumeer/engine/controller/ViewFacade.java

```diff
@@ -59,48 +60,65 @@
    @Inject
    private SecurityFacade securityFacade;
 
+   @Inject
+   private ProjectFacade projectFacade;
+
+   @Inject
+   private OrganisationFacade organisationFacade;
+
    /**
     * Creates initial metadata for the view
     *
     * @param originalViewName
     *       name given by user
     * @param viewType
     *       type of the view
+    * @param description
+    *       view description
     * @param configuration
     *       configuration of the view
     * @return view id
     * @throws ViewAlreadyExistsException
     *       when view with given name already exists
     */
-   public int createView(final String originalViewName, final String viewType, final DataDocument configuration) throws ViewAlreadyExistsException {
-      if (!dataStorage.hasCollection(LumeerConst.View.VIEW_METADATA_COLLECTION_NAME)) {
-         dataStorage.createCollection(LumeerConst.View.VIEW_METADATA_COLLECTION_NAME);
+   public int createView(
+         final String originalViewName,
+         final String viewType,
+         final String description,
+         final DataDocument configuration) throws ViewAlreadyExistsException {
+
+      if (!dataStorage.hasCollection(metadataCollection())) {
+         dataStorage.createCollection(metadataCollection());
```

Comment:
Do we want to check this every time this method is called? Isn't it an unnecessary database call? What about moving the initialization to `DatabaseInitializer`?